iron-selectable.html

```diff
@@ -37,6 +37,13 @@
        */
 
       /**
+       * Fired when the fallback-selection is triggered. Does not require
```

Comment:
I chose the word triggered for the reason stated in the above comment: the fallback selection is not selected in the selector itself, it was only triggered. Therefore I prefer the triggered naming, but if you really prefer the other name I am okay with that too.